spec/services/sqs/publish_laa_reference_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+RSpec.describe Sqs::PublishLaaReference do
+  let(:prosecution_case_id) { '7a0c947e-97b4-4c5a-ae6a-26320afc914d' }
+  let(:defendant_id) { '8cd0ba7e-df89-45a3-8c61-4008a2186d64' }
+  let!(:prosecution_case) do
+    ProsecutionCase.create!(
+      id: prosecution_case_id,
+      body: JSON.parse(file_fixture('prosecution_case_search_result.json').read)
+    )
+  end
+  let(:maat_reference) { 123_456 }
+
+  let(:sqs_payload) do
+    {
+      maatId: 123_456,
```

Comment:
Could this use the `maat_reference` variable defined above? (Or remove that `let` block because it doesn't seem to be being used anywhere else)

Reply:
Updated to use `maat_reference`